src/modules/authentication/api/authentication.controller.ts

```diff
@@ -99,10 +100,17 @@ export class AuthenticationController {
     @ApiUnauthorizedResponse({ description: 'User is not logged in.' })
     @ApiOkResponse({ description: 'Returns info about the logged in user.', type: UserinfoResponse })
     public async info(@Permissions() permissions: PersonPermissions): Promise<UserinfoResponse> {
-        const roleIds: RolleID[] = (await permissions.getRoleIds()).map((item: RolleID) => item);
+        const roleIds: string[] = (await permissions.getRoleIds()).map((item: string) => item);
```

Comment:
Not what I meant, this line can just be
`const roleIds: RolleID[] = (await permissions.getRoleIds());`

Mapping any value to itself does nothing